wsl-pro-service/internal/system/backend.go

```diff
@@ -22,24 +24,29 @@ func (b realBackend) GetenvWslDistroName() string {
 }
 
 // ProExecutable returns the full command to run the pro executable with the provided arguments.
-func (b realBackend) ProExecutable(args ...string) (string, []string) {
-	return "pro", args
+func (b realBackend) ProExecutable(ctx context.Context, args ...string) *exec.Cmd {
+	return exec.CommandContext(ctx, "pro", args...)
 }
 
-func (b realBackend) LandscapeConfigExecutable(args ...string) (string, []string) {
-	return "landscape-config", args
+func (b realBackend) LandscapeConfigExecutable(ctx context.Context, args ...string) *exec.Cmd {
+	return exec.CommandContext(ctx, "landscape-config", args...)
 }
 
 // ProExecutable returns the full command to run the wslpath executable with the provided arguments.
-func (b realBackend) WslpathExecutable(args ...string) (string, []string) {
-	return "wslpath", args
+func (b realBackend) WslpathExecutable(ctx context.Context, args ...string) *exec.Cmd {
+	return exec.CommandContext(ctx, "wslpath", args...)
 }
 
 // WslinfoExecutable returns the full command to run the wslinfo executable with the provided arguments.
-func (b realBackend) WslinfoExecutable(args ...string) (string, []string) {
-	return "wslinfo", args
+func (b realBackend) WslinfoExecutable(ctx context.Context, args ...string) *exec.Cmd {
+	return exec.CommandContext(ctx, "wslinfo", args...)
 }
 
-func (b realBackend) CmdExe(path string, args ...string) (string, []string) {
-	return path, args
+func (b realBackend) CmdExe(ctx context.Context, path string, args ...string) *exec.Cmd {
```

Comment:
Most of this file is inherently uncoverable by package tests, but I guess we could add a small test function in `system_test.go` that asserts the returned `exec.Cmd`object's Dir respects the invariant herein proposed. I'm pretty sure that small test will be enough to please codecov.